test/allowlist_for_publicAPI.json

```diff
@@ -998,6 +998,7 @@
     "Iterable",
     "Iterator",
     "List",
+    "Mapping",
```

Comment:
we shouldn't modify this file, but instead add an `__all__` for `torch.distributed.fsdp.fully_sharded_data_parallel` instead

Reply:
Please re-confirm since all other names imported from `typings` are included in this list. I think even if an `__all__` solution were to be introduced, it should be resolved in another PR since it's irrelevant to the problem addressed here.